src/Documentation/sidebar.json

```diff
@@ -76,6 +76,7 @@
     "files": [
       "add.md",
       "checkout.md",
+      "cache.md",
```

Comment:
move it above checkout )

Reply:
Looks like it is too late for me :slightly_smiling_face:

Reply:
Done. Github doesn't refresh it here for some reason.